frontend/src/modules/alarms/AlarmsPage.tsx

```diff
@@ -287,30 +292,56 @@ export const AlarmsPage = (): JSX.Element => {
   };
 
   const alarmLimitsRequest = useSelector(getAlarmLimitsRequestStandby);
+  const alarmLimitsActual = useSelector(getAlarmLimits);
   const dispatch = useDispatch();
   const currentMode = useSelector(getParametersRequestMode);
   const ventilating = useSelector(getIsVentilating);
   const [alarmLimits, setAlarmLimits] = useState(alarmLimitsRequest as Record<string, Range>);
+  const [alarmLimitsAct] = useState(alarmLimitsActual as Record<string, Range>);
   const updateAlarmLimits = (data: Partial<AlarmLimitsRequest>) => {
     setAlarmLimits({ ...alarmLimits, ...data } as Record<string, Range>);
     dispatch(updateCommittedState(ALARM_LIMITS_STANDBY, alarmLimits));
   };
   const applyChanges = () => dispatch(updateCommittedState(ALARM_LIMITS, alarmLimits));
   const alarmConfig = alarmConfiguration(currentMode);
   const [open, setOpen] = useState(false);
+  const [discardOpen, setDiscardOpen] = useState(false);
+  const [isDisabled, setIsDisabled] = useState(true);
 
   const handleClose = () => {
     setOpen(false);
   };
 
+  const handleDiscardClose = () => {
+    setDiscardOpen(false);
+  };
+
   const handleConfirm = () => {
     setOpen(false);
     applyChanges();
+    setIsDisabled(true);
+  };
+
+  const handleDiscardConfirm = () => {
+    setDiscardOpen(false);
+    handleDiscard();
+    setIsDisabled(true);
   };
+
+  const handleDiscard = () => {
+    const storeData = store.getState();
```

Comment:
Are you directly accessing the store because it won't work to use the `getAlarmLimits` selector?